packages/client/src/forms/register/reducer.ts

```diff
@@ -46,9 +52,17 @@ export const registerFormReducer: LoopReducer<IRegisterFormState, Action> = (
 ): IRegisterFormState | Loop<IRegisterFormState, Action> => {
   switch (action.type) {
     case offlineActions.READY:
-    case offlineActions.DEFINITIONS_LOADED:
-      const birth = deserializeForm(action.payload.forms.registerForm.birth)
-      const death = deserializeForm(action.payload.forms.registerForm.death)
+    case offlineActions.CONTENT_LOADED:
+      const birth = deserializeForm(
+        process.env.NODE_ENV === 'test'
+          ? (mockOfflineData.forms.registerForm.birth as ISerializedForm)
+          : (registerForms.birth as ISerializedForm)
+      )
+      const death = deserializeForm(
+        process.env.NODE_ENV === 'test'
+          ? (mockOfflineData.forms.registerForm.death as ISerializedForm)
+          : (registerForms.death as ISerializedForm)
+      )
```

Comment:
Not entirely sure to be honest, but I having code paths only for tests is a bit of a bad practice. Would it be possible to mock the `@client/forms/register/fieldDefinitions/register` module in tests and output the mockOfflineData forms as `registerForms` from there?

Reply:
I gave it a try so that I moved mockOfflineData to it's own file under `/tests`, then in setupTests did

```ts
import {
  mockOfflineData
} from './tests/mock-offline-data'


jest.mock('@client/forms/register/fieldDefinitions/register', () => ({
  registerForms: mockOfflineData.forms.registerForm
}))
```

Seems to work fine 👍